src/actions/node-type.js

```diff
@@ -1,14 +1,22 @@
 export const TOGGLE_TYPE_DISABLED = 'TOGGLE_TYPE_DISABLED';
 
 /**
- * Toggle a node-type's visibility on/off
- * @param {string} typeID Type id
- * @param {Boolean} disabled True if type is disabled
+ * The default enabled / disabled value for all types is 'unset', meaning not explicitly set by user.
+ * In practice 'unset' acts like enabled but is a distinct state for UI purposes.
```

Comment:
I am slightly confused about the wording here, in particular the  `UI purpose` for `unset` - is this action type particularly used to cater for unsetting all element type filters? It would be nice to further elaborate on what the `distinct state for UI purposes` actually us.